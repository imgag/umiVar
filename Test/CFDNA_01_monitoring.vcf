##fileformat=VCFv4.1
##fileDate=13/11/2020
#CHROM	POS	ID	REF	ALT	QUAL	FILTER
chr1	47395874	M	A	C	.	PASS
chr1	115256530	M	G	T	.	PASS
chr1	160786670	M	A	G	.	PASS
chr1	233448413	ID	C	T	.	PASS
chr11	5709028	ID	C	T	.	PASS
