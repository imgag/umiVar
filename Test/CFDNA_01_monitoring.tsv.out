CHROM	POS	REF	ALT	DP_HQ	REF_COUNT	ALT_COUNT	AF	Multi_REF	Multi_ALT	Multi_AF	Pval	FDR	STRAND	FISHER	OAAC	MAL	Sequence_Context	Homopolymer	FILTER
chr1	47395874	A	C	6730	6716	11	0.001634	4749	8	0.001682	1.69e-03	2.61e-01	2-9-3370-3346	1.00e+00	3	1.00e+00	CGTGCAATGGG	0	FDR
chr1	115256530	G	T	12276	11962	311	0.025334	6974	165	0.023112	3.05e-10	1.53e-07	272-39-9299-2663	4.21e-02	3	1.00e+00	TTCTTGTCCAG	0	PASS
chr1	160786670	A	G	5551	4267	1277	0.230049	3038	927	0.233796	1.76e-39	1.77e-36	648-629-2121-2146	1.00e+00	7	1.00e+00	TCTCTATTTAC	0	PASS
